package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "galoy",
   "scripts": {
-    "tsc-check": "tsc --noEmit -p tsconfig.d.json && tsc --noEmit --skipLibCheck",
+    "tsc-check": "tsc --noEmit -p tsconfig.d.json --skipLibCheck && tsc --noEmit --skipLibCheck",
```

Comment:
We can't skipLibCheck for the declaration files check. I'll look into the errors

Reply:
we can't or don't want to? for me the diff here solve the issue but I'm not sure if we're supposed to do that

Reply:
That task is used to check our declaration files (which `skipLibCheck` skips).

It seems you have some extra packages in your node_modules folder, nuke and retry please

Reply:
had a nodes_modules in parent folder that was messing things up... closing this now